pkg/cmd/target/remove.go

```diff
@@ -19,35 +19,43 @@ import (
 var targetRemoveCmd = &cobra.Command{
 	Use:     "remove",
```

Comment:
Please add `[TARGET_NAME]` to the end to make it more clear to users.

Reply:
@tarunrajput I think you may have overridden the change as it's just `"remove"` again.

Reply:
ah! my bad. I've added it now.